src/handlers/query-user.ts

```diff
@@ -0,0 +1,62 @@
+import { Context } from "../types/context";
+
+export async function queryUser(context: Context, username: string) {
+  const {
+    octokit,
+    payload,
+    adapters: { supabase },
+    config,
+  } = context;
+  const body: string[] = [];
+  try {
+    const {
+      data: { id },
+    } = await octokit.users.getByUsername({
+      username,
+    });
+    if (!config.allowPublicQuery && payload.organization) {
+      const { status } = await octokit.orgs.checkMembershipForUser({
+        username,
+        org: payload.organization.login,
+      });
+      // @ts-expect-error Somehow typing seems wrong but according to
+      // https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#check-organization-membership-for-a-user--status-codes
+      // 204 means the user is part of the Organization
+      if (status !== 204) {
+        body.push(`\`\`\`User ${username} cannot request another user as it is not member of the organization.\`\`\``);
+        await octokit.issues.createComment({
+          body: body.join("\n"),
+          owner: payload.repository.owner.login,
+          repo: payload.repository.name,
+          issue_number: payload.issue.number,
+        });
+        return;
+      }
+    }
+    const access = await supabase.access.getAccess(id);
+    const wallet = await supabase.wallet.getWallet(id);
+    if (!access && !wallet) {
+      body.push(`\`\`\`
+User information for ${username} was not found.
+\`\`\``);
+    } else {
+      body.push(`
+| Property | Value |
+|----------|-------|`);
+      if (wallet) {
+        body.push(`| Wallet | ${wallet.address} |`);
+      }
+      if (access) {
+        body.push(`| Access | ${access.multiplier_reason} |`);
```

Comment:
It's supposed to store a JSON object of key value pairs. We should be able to write any key/value.

Reply:
Access data comes from there
https://github.com/ubiquity/ubiquibot/blob/579c447ce23397d8947e2f3f2a5e665f1c60fedf/src/handlers/comment/handlers/query.ts#L42

And the current schema seems to be
<img width="722" alt="image" src="https://github.com/ubiquibot/command-query-user/assets/9807008/03160a8f-a54a-46b8-a98c-ed7ee61c26be">

This being said, table is empty and never had to use it before. I just put it there because of the old code, I can remove it if it has no use.

Reply:
We don't really use `multiplier` anymore that's true but one of the first questions that powerhouse asked related to the capabilities of `multiplier` so I feel like we should not neglect this capability as it may find more use in other organizations